tests/port-forwarding.go

```diff
@@ -28,6 +28,17 @@ type portForwarderImpl struct {
 
 var _ PortForwarder = &portForwarderImpl{}
 
+type portForwardConn struct {
+	net.Conn
+	streamConnCloser io.Closer
+}
+
+func (p *portForwardConn) Close() error {
+	err1 := p.Conn.Close()
```

Comment:
Inline?

Reply:
I wanted to make sure that the first connection was closed before the second one. But I see that golang defines the order of evaluation of function arguments, unlike other languages, where it is up to the compiler optimization.
https://go.dev/ref/spec#Order_of_evaluation

Reply:
done